.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**H2O version, Operating System**
+(e.g. 3.38.0.1, Mac OS w/ M1 chip)
+
```

Comment:
I would ask for more details about the environment. Standalone H2O/K8s/Hadoop, version of Hadoop distribution. Version of Python/R.